modules/etcd-increase-db.adoc

```diff
@@ -12,7 +12,9 @@ You might want to increase the disk quota if you encounter a `low space` alert.
 
 Another scenario where you might want to increase the disk quota is if you encounter an `excessive database growth` alert. This alert is a warning that the database might grow too large in the next four hours. In this scenario, consider increasing the disk quota so that you do not eventually encounter a `low space` alert and possible write fails. 
 
-When you increase the disk quota, the amount of space that you specify is not immediately reserved. Instead, etcd can grow to that size if needed. Ensure that etcd is running on a dedicated disk that is larger than the value that you specify for the disk quota.
+When you increase the disk quota, the amount of space that you specify is not immediately reserved. Instead, etcd can grow to that size if needed. Ensure that etcd is running on a dedicated disk that is larger than the value that you specify for the disk quota. 
+
+For large etcd databases, the control plane nodes must have extra memory and storage. Because you need to account for the API server cache, the minimum memory required is 3 times the size of the etcd database.
```

Comment:
@lahinson Could we edit this to say 3 times the configured size of the database. I first read this as 3x the current size; not the maximum.

Reply:
@dusk125 Good suggestion. Should it be "3 times the configured size of the database" or "3 times the size of the configured etcd database"?

Reply:
Thanks! Just made that revision.

Reply:
/lgtm